### DISCUSSION TITLE
How to add text to plot at given points?

### DISCUSSION THREAD

**Discussion by Agam9427** [highfestiva/finplot#434] [General]
How do I add text to a plot for a given (x,y) coordinate on the graph?

**Comment by highfestiva** [comment]
There are two examples in the `examples/` directory: one for BitFinex (bfx short), and the other one called pandas-pd-plot or something like that.

**Comment by armeya25** [comment]
X is datetime object /index
Y is value at which you want to draw

Fplt.add_text((x,y), color='black')

**Reply by highfestiva** [reply]
You're right, that is in the line.py example.
